frameworks/youth-risk-assessment/questions/care-information.yml

```diff
@@ -0,0 +1,45 @@
+type: checkbox
+question: Is there any information that will help with caring for this young person during the move?
```

Comment:
I've noticed we use a mixture of `young person` and `they/their`. We should probably take a consistent approach?

Reply:
**Agreed during call:** consistently use `they/their` in question and `this person` in more detail error message.